src/main/resources/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder/config.jelly

```diff
@@ -167,7 +204,19 @@
                         select.trigger('change');
                     }
                 }
+            }
 
+            switcherTestInfoLastConfiguration_${descriptor.defaultBuildID()} = function(){
+          	    //selected input type, from last configuration
+          	    var lastSwitcherConfig = '${instance.getInputTestInfoSwitcher()}';
+
+          	    if(lastSwitcherConfig !== '' ){//making sure the switcher gets the last configuration value
+          	    	var select = Q("#selectTestInfoInputType_${descriptor.defaultBuildID()}");
+          	    	if(select){
```

Comment:
This select will always be a jQuery object, so it will always be evaluated to true. If you want to check if the _select_ was found, please check the length instead
`if (select.length > 0){`